app/boards/arm/nice_nano/nice_nano.yaml

```diff
@@ -13,3 +13,6 @@ supported:
   - ieee802154
   - pwm
   - watchdog
+manufacturer: Nice Keyboards
+compatible: Pro Micro
```

Comment:
Is the idea that boards and shields each include a `compatible` value, and if they match, the can work together?

For a few edge case boards, we may need to make the `compatible` an array. I'm thinking of the Pancake v1, which supports both pro micro and feather footprints.

Reply:
This is the idea, yeah. I think an array makes sense, although I wonder if I can support both, as having the array syntax in every yaml seems a bit annoying when only a few boards out there support multiple interconnects.